css/style.css

```diff
@@ -0,0 +1,195 @@
+body {
+  /* font-size: 40px; */
+  font-family: fantasy;
+  color: rgb(44, 44, 44);
+}
+/* Icons styling */
```

Comment:
But this comment is fine - because you're describing what code/styles below will relate to.